features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/TimelineView.kt

```diff
@@ -165,6 +170,10 @@ fun TimelineItemRow(
                 onClick(timelineItem)
             }
 
+            fun onSwipeToReply() {
+                onSwipeToReply(timelineItem)
```

Comment:
This can lead to issues with cached values for `timelineItem`. The `timelineItem` returned here is not the one from the last recomposition, but from the 1st one, when this function is assigned, so if it's later edited or redacted we'll be reporting the wrong value on the callback. I had to revert `onClick` and `onLongClick` on https://github.com/vector-im/element-x-android/pull/771 for this same reason.

Reply:
Ah, good point, thanks for the feedback. I will change that.

Reply:
218002627d9397ec83534b08da1437b800b0bf8a